apps/src/templates/studioHomepages/CourseBlocksTools.jsx

```diff
@@ -5,52 +5,63 @@ import ContentContainer from '../ContentContainer';
 import ResourceCard from './ResourceCard';
 import ResourceCardResponsiveContainer from './ResourceCardResponsiveContainer';
 import i18n from '@cdo/locale';
-import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import {pegasus, studio} from '@cdo/apps/lib/util/urlHelpers';
 
 class CourseBlocksTools extends Component {
   static propTypes = {
-    isEnglish: PropTypes.bool.isRequired
+    isEnglish: PropTypes.bool.isRequired,
+    showAiCard: PropTypes.bool
   };
 
-  cards = [
-    {
-      heading: i18n.courseBlocksToolsAppLab(),
-      description: i18n.courseBlocksToolsAppLabDescription(),
-      path: 'applab'
-    },
-    {
-      heading: i18n.courseBlocksToolsGameLab(),
-      description: i18n.courseBlocksToolsGameLabDescription(),
-      path: 'gamelab'
-    },
-    {
-      heading: i18n.courseBlocksToolsWebLab(),
-      callout: `(${i18n.beta()})`,
-      description: i18n.courseBlocksToolsWebLabDescription(),
-      path: 'weblab'
-    },
-    {
-      heading: i18n.csJourneys(),
-      callout: i18n.newExclame(),
-      description: i18n.csJourneysDescription(),
-      path: 'csjourneys'
-    },
-    {
-      heading: i18n.courseBlocksToolsVideo(),
-      description: i18n.courseBlocksToolsVideoDescription(),
-      path: 'videos'
-    },
-    {
-      heading: i18n.courseBlocksToolsWidgets(),
-      description: i18n.courseBlocksToolsWidgetsDescription(),
-      path: 'widgets'
+  constructor(props) {
+    super(props);
+    this.cards = [
+      {
+        heading: i18n.courseBlocksToolsAppLab(),
+        description: i18n.courseBlocksToolsAppLabDescription(),
+        link: pegasus('/applab')
+      },
+      {
+        heading: i18n.courseBlocksToolsGameLab(),
+        description: i18n.courseBlocksToolsGameLabDescription(),
+        link: pegasus('/gamelab')
+      },
+      {
+        heading: i18n.courseBlocksToolsWebLab(),
+        callout: `(${i18n.beta()})`,
+        description: i18n.courseBlocksToolsWebLabDescription(),
+        link: pegasus('/weblab')
+      },
+      {
+        heading: i18n.csJourneys(),
+        callout: i18n.newExclame(),
+        description: i18n.csJourneysDescription(),
+        link: pegasus('/csjourneys')
+      },
+      {
+        heading: i18n.courseBlocksToolsVideo(),
+        description: i18n.courseBlocksToolsVideoDescription(),
+        link: pegasus('/videos')
+      }
+    ];
+    if (props.isEnglish && props.showAiCard) {
+      this.cards.push({
+        heading: i18n.courseBlocksToolsAi(),
+        callout: i18n.newExclame(),
+        description: i18n.courseBlocksToolsAiDescription(),
+        link: studio('/s/aiml-2021')
```

Comment:
Interesting.  Any color on why aiml doesn't have a landing page on pegasus like the other cards?

Reply:
there's a whole[ slack thread](https://codedotorg.slack.com/archives/C06E60KL4/p1625159465045400) about it!